lib/ruby_lsp/requests/code_lens.rb

```diff
@@ -86,11 +99,25 @@ def visit_vcall(node)
 
       private
 
-      sig { params(node: SyntaxTree::Node, path: String, name: String, command: String).void }
-      def add_code_lens(node, path:, name:, command:)
+      sig { params(node: SyntaxTree::Node, name: String, command: String).void }
+      def add_code_lens(node, name:, command:)
         @results << Interface::CodeLens.new(
           range: range_from_syntax_tree_node(node),
-          command: Interface::Command.new(title: "Run", command: "rubyLsp.runTest", arguments: [path, name, command]),
+          command: Interface::Command.new(
+            title: "Run",
+            command: "rubyLsp.runTest",
+            arguments: [
+              @path,
+              name,
+              command,
+              {
+                start_line: node.location.start_line,
+                start_char: node.location.start_char,
+                end_line: node.location.end_line,
+                end_char: node.location.end_char,
+              },
```

Comment:
To create a [`TestItem`](https://code.visualstudio.com/api/references/vscode-api#TestItem) in the Test Explorer, you need to supply a [`Range`](https://code.visualstudio.com/api/references/vscode-api#Range). Ranges need a `start_line`, `start_char`, `end_line`, and `end_char` so this info is necessary.

Reply:
In this case, we probably need to adjust lines by 1, since Syntax Tree is not zero based. And we probably want the column instead of the absolute character.

```suggestion
              {
                start_line: node.location.start_line - 1,
                start_column: node.location.start_column,
                end_line: node.location.end_line - 1,
                end_column: node.location.end_column,
              },
```